components/03-organisms/site-footer/yds-site-footer.twig

```diff
@@ -16,61 +16,18 @@
   'data-component-width': 'site',
   'data-site-footer-border-thickness': site_footer__border_thickness|default('0'),
   'data-footer-theme': site_footer__theme|default('one'),
+  'data-footer-variation': site_footer__variation|default('basic'),
   class: bem(site_footer__base_class),
 } %}
 
 <footer {{ add_attributes(site_footer__attributes) }}>
   <div {{ bem('inner', [], site_footer__base_class) }}>
-    {# Primary #}
-    <div {{ bem('primary', [], site_footer__base_class) }}>
-      {# Branding #}
-      <div {{ bem('branding', [], site_footer__base_class) }}>
-        <div {{ bem('logo', [], site_footer__base_class) }}>
-          <a data-link-style="no-underline" class="site-footer__site-branding site-footer__site-branding--primary" href="https://yale.edu">&#XF2A2;</a>
-        </div>
-        {% block site_footer__text %}
-          {% include "@page-layouts/placeholder/yds-placeholder.twig" with {
-            placeholder: 'Text',
-            placeholder__type: 'element',
-          } %}
-        {% endblock %}
-      </div>
-      {# Columns #}
-      <div {{ bem('columns', [], site_footer__base_class) }}>
-        {% block site_footer__columns %}
-          {% include "@page-layouts/placeholder/yds-placeholder.twig" with {
-            placeholder: 'Columns',
-            placeholder__type: 'element',
-          } %}
-        {% endblock %}
-      </div>
-    </div>
-    {# Secondary #}
-    <div {{ bem('secondary', [], site_footer__base_class) }}>
-      {# Social #}
-      <div {{ bem('social', [], site_footer__base_class) }}>
-        {% block site_footer__social %}
-          {% include "@molecules/social-links/yds-social-links.twig" %}
-        {% endblock %}
-      </div>
-      {# Meta #}
-      <div {{ bem('meta', [], site_footer__base_class) }}>
-        {% include "@atoms/controls/text-link/yds-text-link.twig" with {
-          link__content: 'Accessibility at Yale',
-          link__url: "https://usability.yale.edu/web-accessibility/accessibility-yale",
-          link__style: 'no-underline',
-          link__modifiers: ['footer-link'],
-        } %}
-        <span {{ bem('divider', [], site_footer__base_class) }}>|</span>
-        {% include "@atoms/controls/text-link/yds-text-link.twig" with {
-          link__content: 'Privacy Policy',
-          link__url: "https://privacy.yale.edu/resources/privacy-statement",
-          link__style: 'no-underline',
-          link__modifiers: ['footer-link'],
-        } %}
-        <span {{ bem('divider', [], site_footer__base_class) }}>|</span>
-        <span>Copyright © {{ 'now'|date('Y') }} Yale University. All rights reserved.</span>
-      </div>
-    </div>
+    {% block footer__inner %}
+      {% if site_footer__variation == 'mega' %}
+        {% include "@organisms/site-footer/_site-footer-mega.twig" %}
+      {% else %}
+        {% include "@organisms/site-footer/_site-footer-basic.twig" %}
+      {% endif %}
+    {% endblock %}
```

Comment:
@nJim Do you feel as though this is enough of an improvement? 

This at least establishes a partial structure should we have a third footer type in the future. This still necessitates conditional logic in the Drupal template at `web/profiles/custom/yalesites_profile/modules/custom/ys_core/templates/ys-footer-block.html.twig`

Reply:
Sorry for missing this comment. I think this add a ton of clarity. TY.